docs/tutorial/intro_to_zydeco.md

```diff
@@ -0,0 +1,168 @@
+# Intro to Zydeco
+## Types
+Zydeco has two main types: value type (`A`) and computation type (`B`). 
+
+Value type can be: 
+- `X`: type variable defined using `data`.
+- `Int | String | Boolean | Unit`: basic primitive type with built-in functions.
+- `Thunk(B)`: suspend a computation type and consider it as a value type. 
+
+Computation type can be:
+- `X`: type variable defined using `codata`.
```

Comment:
In line with the `A/B` convention I would use `X` for value type variables and `Y` for computation type variables.